act 3, act 4 번역완료, 첨삭 필요

Skygazer-Jay opened the pull request:
act 3 번역 완료했습니다만... 그 외 나머지 변경내역은 왜 있는지 모르겠네요;;
첨삭 부탁드립니다! 

그나저나 번역본이 어떻게 나오는지 보려는데 실행이 안되네요 (로딩 중간에서 걸립니다.) 어떻게 해야 할까요?;;

KyleHeren committed:
Update PROMO.md

KyleHeren committed:
Update act1.md

KyleHeren committed:
Update act1.md

KyleHeren committed:
Update act1.md

ncase committed:
actually final fixes aaaahhhh

KUMAbabe committed:
인트로 업데이트

// 부분은 필요 시 수정을 위하여 남겨 둔 원문입니다
혹시 모를 코딩 오류를 위하여 다른 부분은 수정하지 않았습니다

KUMAbabe committed:
게임 클리어

클리어 시 결과 발표 및 자동저장 완료 메세지

KUMAbabe committed:
9/26 1화 수정중

지금 게임 플레이하면서 하고 있는데 루트가 엄청나게 많습니다
이걸 4화까지 반복할 생각하니 ㅎㅎ
아 정말 신난다
원문에는 "//" 표시를 했습니다
검토 및 첨삭 부탁드립니다

KUMAbabe committed:
Update README.md

KUMAbabe committed:
Merge pull request #1 from KUMAbabe/KUMAbabe-patch-1

Update README.md

KUMAbabe committed:
Update intro.md

KUMAbabe committed:
Merge pull request #2 from KUMAbabe/KUMAbabe-patch-2

Update intro.md

KUMAbabe committed:
Merge pull request #3 from KUMAbabe/patch-1

인트로

KUMAbabe committed:
Merge pull request #4 from KUMAbabe/patch-2

게임 클리어

KUMAbabe committed:
9/25 1화 수정

9/26 1화 수정중

KUMAbabe committed:
Update README.md

KUMAbabe committed:
Update intro.md

KUMAbabe committed:
인트로 (첨삭 필요)

게임 시작 및 주인공의 빨간 후드가 늑대로 변하기 전까지의 인트로 장면입니다

KUMAbabe committed:
9/26 테스트

테스트

KUMAbabe committed:
테스트 패치1

테스트 패치 1

KUMAbabe committed:
1화 완료(첨삭 필요)

"//" 친 부분은 원문

KUMAbabe committed:
게임 인터페이스

html 타이틀 및 구석마다 있는 텍스트 번역 완료
사실 번역 완료되면 209~211 라인은 없애도 됨
그래도 후에 무슨 일이 있을 지 몰라서 그냥 놔둠

KUMAbabe committed:
1화 번역 완료(첨삭 필요)

"//" 부분은 원문
첨삭 완료 후 삭제 필수

KUMAbabe committed:
9/27 2화 진행중

1화보다는 루트 적음
길어야 이틀 걸릴 듯

KUMAbabe committed:
html 인터페이스 및 인덱스

대충 보이는 것만 번역했습니다
사실 번역 완료되면 209~211 라인은 없애도 되는데, 아직은 아니니까 그냥 둘게요.

KUMAbabe committed:
9/27 2화 진행중

9/27 2화 진행중
1화보다는 루트 적어서 할만함

KUMAbabe committed:
Merge branch 'gh-pages' into master

KUMAbabe committed:
옵션, About, 개발 노트

KUMAbabe committed:
Merge branch 'master' into KUMAbabe-patch-html2

KUMAbabe committed:
옵션, About, 개발 노트

옵션, About, 개발 노트

KUMAbabe committed:
Update index.html

KUMAbabe committed:
한글깨짐 1트 도전

제발 돼라

KUMAbabe committed:
인터페이스 한글깨짐 2트

인터페이스 한글깨짐 2트

KUMAbabe committed:
인터페이스 한글깨짐 2트

인터페이스 한글깨짐 2트

KUMAbabe committed:
인터페이스 한글깨짐 3트

인터페이스 한글깨짐 3트

KUMAbabe committed:
인터페이스 한글깨짐 3트

인터페이스 한글깨짐 3트

KUMAbabe committed:
한글 나온다

만쉐이

KUMAbabe committed:
한글 나온다

만쉐이

KUMAbabe committed:
Update index.html

KUMAbabe committed:
Merge branch 'master' of https://github.com/KUMAbabe/anxiety

KUMAbabe committed:
옵션 텍스트

제발 돼라

KUMAbabe committed:
옵션 텍스트

옵션 텍스트

KUMAbabe committed:
Update Options.js

KUMAbabe committed:
옵션

옵션 수정

Skygazer-Jay committed:
Add files via upload

Skygazer-Jay committed:
Add files via upload

Skygazer-Jay committed:
Update act3.md

Skygazer-Jay committed:
Delete act3.md

KUMAbabe commented:
> 그나저나 번역본이 어떻게 나오는지 보려는데 실행이 안되네요 (로딩 중간에서 걸립니다.) 어떻게 해야 할까요?;;

저도 계속 알아보고 있는데 잘 안 되네요
intro에서 막히니 원;;;
익스플로러, 엣지, 크롬, 파폭 전부 다 안되는 상황입니다

Skygazer-Jay commented:
> > 그나저나 번역본이 어떻게 나오는지 보려는데 실행이 안되네요 (로딩 중간에서 걸립니다.) 어떻게 해야 할까요?;;
> 
> 저도 계속 알아보고 있는데 잘 안 되네요
> intro에서 막히니 원;;;
> 익스플로러, 엣지, 크롬, 파폭 전부 다 안되는 상황입니다

번역본 말고 원본마저도 ZIP로 다운받아서 실행하면 오페라는 로딩도 다 안되고, 엣지는 로딩까지는 되지만 인트로를 넘어가지 않네요.
act1.md의 {{if window.localStorage.continueChapter=="replay"}}이후로 안되는 거 같은데 해결방안이 없으려나요

KUMAbabe commented:
> 번역본 말고 원본마저도 ZIP로 다운받아서 실행하면 오페라는 로딩도 다 안되고, 엣지는 로딩까지는 되지만 인트로를 넘어가지 않네요.
> act1.md의 {{if window.localStorage.continueChapter=="replay"}}이후로 안되는 거 같은데 해결방안이 없으려나요

야매로 강제로 구동시키는 방법을 issue에 올려놨습니다
야매이기 때문에 작동은 정상적으로 되진 않는다는 점 미리 염두에 두시면 될 거 같습니다
내일이나 내일 모레 쯤 해서 저도 계속 알아보겠습니다

Skygazer-Jay commented:
> > 번역본 말고 원본마저도 ZIP로 다운받아서 실행하면 오페라는 로딩도 다 안되고, 엣지는 로딩까지는 되지만 인트로를 넘어가지 않네요.
> > act1.md의 {{if window.localStorage.continueChapter=="replay"}}이후로 안되는 거 같은데 해결방안이 없으려나요
> 
> 야매로 강제로 구동시키는 방법을 issue에 올려놨습니다
> 야매이기 때문에 작동은 정상적으로 되진 않는다는 점 미리 염두에 두시면 될 거 같습니다
> 내일이나 내일 모레 쯤 해서 저도 계속 알아보겠습니다

늦은 시각에 죄송합니다. 

![wut](https://user-images.githubusercontent.com/55949965/65904034-8d16ff80-e3f8-11e9-85f6-cfe51ece633e.PNG)
엣지에서 원본ZIP 다운 받아서 실행해 보았는데 인트로에서 막힐 때 요소 검사의 콘솔창에 아래 두 에러가 뜨더군요 

뭘 의미하는지는 프로그래밍알못이라;; 어쨌든 단서가 될 법해서 올려봅니다.

오페라에서 돌릴 때는 CORS policy 에러 때문에 로딩이 완전히 되지 않는 것 같습니다.

KUMAbabe commented:
네 감사합니다. 
그, 저도 방금 issue에서 봤는데요
콘솔에서 Game.debug() 를 치면 디버그 모드가 나온다고 합니다
저도 엣지로 해 보니 이렇게 나오네요
번역하신 내용 확인용으로 사용하셔도 될 거 같습니다

![0](https://user-images.githubusercontent.com/51105929/65906193-06185600-e3fd-11e9-8b7e-7fed1c08b434.PNG)

Skygazer-Jay committed:
Add files via upload

Skygazer-Jay committed:
Update act4.md

Skygazer-Jay commented:
콘솔 디버그는 했습니다만 대사들이 위로 올라가지 않고 화면 아래로 쌓이면서 내려가서 모든 대사를 확인 하긴 어렵더군요 ㅜㅜ 대체 뭐가 문제일까요...

KUMAbabe committed:
// 삭제

쥰내 길어서 그냥 삭제함

KUMAbabe committed:
// 삭제, 계속 진행중

초벌은 이거 하나 남았음

KUMAbabe committed:
한글화한 스프라이트

Patreon 하나만 하면 되는데
오늘은 그냥 자야지

KUMAbabe committed:
저게 어떻게 "읽는 걸 좋아하시나요" 냐 멍충아

으이구 영어 공부 좀 하자

KyleHeren commented:
KUMABabe 님과 Skygazer-Jay 님의 pull request 를 확인했습니다! 몇 군데 수정할 부분이 보이는데, 어떻게 수정하는지를 모르겠네요 :( 혹시 알려주실 수 있으신가요...?

파일을 가져오는 건 Merge pull request 를 하면 되는 건가요? 그것도 알려주세요 D:

Skygazer-Jay commented:
> KUMABabe 님과 Skygazer-Jay 님의 pull request 를 확인했습니다! 몇 군데 수정할 부분이 보이는데, 어떻게 수정하는지를 모르겠네요 :( 혹시 알려주실 수 있으신가요...?
> 
> 파일을 가져오는 건 Merge pull request 를 하면 되는 건가요? 그것도 알려주세요 D:

저도 처음이라 잘 모르겠습니다만 Merge하면 Kyle님 포크랑 합쳐지니까 수정할 수 있게 되지 않을까요..?

Skygazer-Jay committed:
Merge branch 'gh-pages' into master

Skygazer-Jay committed:
Merge pull request #1 from KUMAbabe/master

한글화 index, 스프라이트, act1, act2(얘만 미완)

Skygazer-Jay committed:
Update act1.md

Skygazer-Jay committed:
Update act1.md

KUMAbabe committed:
Merge branch 'gh-pages' into master

Skygazer-Jay committed:
act1, act2 번역/ 볼드화 완료. act1 실험용

Skygazer-Jay committed:
Update act1.md

138 실험용

Skygazer-Jay committed:
//들 제거. (볼드체는 적용이 안되나?)

Skygazer-Jay committed:
//제거

Skygazer-Jay committed:
//제거

Skygazer-Jay committed:
실험용 스크립트, 243

Skygazer-Jay committed:
결론: 볼드체는 적용 안됨. 선택지 문제 미해결

KUMAbabe committed:
Skygazer-Jay님의 혼이 들어간 번역본

아직 손볼 곳 조금 있음

KUMAbabe committed:
다시 원본

근ㅡ본으로 돌아갔는데도 안 되는 거면
다른 곳에 문제 있는 거임

KUMAbabe committed:
다른 거 안 건드리고 텍스트만 바꿈

이랬는데도 안 되면 그때부터는 코드 만지작거려야 됨
최대한 코드 쪽은 피할 수 있도록
솔직히 남의 코드 보는 게 쥰내 머리아픔

KUMAbabe committed:
구동 확인 완료

이거 띄어쓰기랑 줄바꾸기 하나만 해도 작동 희한하게 됨
주의 요망

KUMAbabe committed:
구동 확인중

원래 act2 하고 있었는데 우리 주인님 여름띠가 개판쳐놔서 순간 이성 잃을 뻔
겨우 붙잡고 act3 올림. 선택지 잘 나오는지 확인
안 나오면 뭐....또 노가다 해야지

KyleHeren commented:
한 부분은 Pull 완료했지만 conflict 가 있어 이 commit 은 pull 이 쉽게 안 되네요... 
![image](https://user-images.githubusercontent.com/55465826/66135221-9096ca80-e634-11e9-8b46-9c0ad3ae1e05.png) 이 화면에서 어떻게 수정을 해야 pull 할 수 있는 건가요?

KUMAbabe committed:
된다!!! 앙~~~기머쩨~~~!!!!

이래놓고 안 되면 레게노ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

KUMAbabe committed:
된다!!! 앙~~~기머쩨~~~!!!!

이래놓고 안 되면 레게노ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

KUMAbabe committed:
act1 2 클리어

KUMAbabe committed:
거의 다 된 거 같은데

줄바꿈 저거 어케 해야 되냐 진짜 어우
폰트 조정도 좀 해야 될 거 같은데

Skygazer-Jay committed:
Merge branch 'gh-pages' into master

Skygazer-Jay committed:
Merge pull request #2 from KUMAbabe/master

자잘한 번역 수정  


'얉다'라니 이게 무슨소리야

Skygazer-Jay committed:
Update act4.md

Skygazer-Jay committed:
Update act1.md

Skygazer-Jay committed:
원본과 스크립트와  대조 일치 확인.

Skygazer-Jay committed:
타언어 포탈(?)

번역해놓고 번역해보지 않을래요라고 하는 건 이상하지 않을깝쇼

Skygazer-Jay committed:
Update act1.md

Skygazer-Jay committed:
Update act2.md

Skygazer-Jay committed:
Update act1.md

Skygazer-Jay committed:
Update act2.md

Skygazer-Jay committed:
Update act2.md

Skygazer-Jay committed:
Update act1.md

Skygazer-Jay committed:
Update act4.md

Skygazer-Jay committed:
Update act4.md
